topology/pre-process-object.c

```diff
@@ -1054,7 +1072,8 @@ static const struct build_function_map *tplg_object_get_map(struct tplg_pre_proc
 			return &object_build_map[i];
 
 		if (!strcmp(class_type, "Dai") &&
-		    !strcmp(object_build_map[i].class_type, "Dai"))
+		    !strcmp(object_build_map[i].class_type, "Dai")  &&
+			!strcmp(object_build_map[i].class_name, ""))
```

Comment:
is this commit separated on purpose or could it be squashed with the previous? I don't see what this brings as a separate patch, or I don't have the background to understand it.

Reply:
yeah, it is supposed to be separated. This is the part where I "tap" into @ranj063 's topology2 pre-processing. So without this none of the dmic/ssp bananas is used. So I wanted it to be separate commit so we could for example set in under some flag -> ok, do generate the nhlt blob or don't do it.

Reply:
I now squashed the commits